js/src/Ice/ObjectAdapter.js

```diff
@@ -29,6 +29,11 @@ import { Debug } from "./Debug.js";
 const _suffixes = [
     "ACM",
     "AdapterId",
+    "Connection.CloseTimeout",
```

Comment:
Should we remove most of these properties from JS, and not add new ones?

Reply:
I removed the new ones, will review the others in a separate PR.